test/catch2/live_restore/live_restore_test_env.h

```diff
@@ -0,0 +1,41 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#pragma once
+
+#include <catch2/catch.hpp>
+#include "../utils.h"
+
+extern "C" {
+#include "wt_internal.h"
+#include "test_util.h"
+#include "../../../../src/live_restore/live_restore_private.h"
+}
+
+#include "../wrappers/connection_wrapper.h"
+
+namespace utils {
+
+class live_restore_test_env {
+public:
+    const std::string _DB_DEST = "WT_LR_DEST";
```

Comment:
Public vars don't need the leading `_` we did away with that accidental style guide hiccup a few months back.

Reply:
[bf5b87d](https://github.com/wiredtiger/wiredtiger/pull/11332/commits/bf5b87d6c93b5e22bfca6554411d14520687bf85)